packages/next/taskfile.js

```diff
@@ -1356,8 +1547,16 @@ export async function ncc(task, opts) {
     )
   await task.parallel(['ncc_webpack_bundle_packages'], opts)
   await task.parallel(['ncc_babel_bundle_packages'], opts)
-  await task.parallel(['copy_constants_browserify'])
-  await task.parallel(['copy_react_server_dom_webpack'])
+  await task.serial(
+    [
+      'ncc_next__react_dev_overlay',
+      'copy_regenerator_runtime',
+      'copy_babel_runtime',
+      'copy_constants_browserify',
+      'copy_react_server_dom_webpack',
+    ],
```

Comment:
Are these serial because each one depends on the previous?

Reply:
It seems when they are parallel taskr outputs them incorrectly